features/CMakeLists.txt

```diff
@@ -1,11 +1,5 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-mbed_add_cmake_directory_if_labels("FEATURE")
-
+add_subdirectory(frameworks/COMPONENT_FPGA_CI_TEST_SHIELD)
```

Comment:
As features is added in the main CMake with EXCLUDE_FROM_ALL, this would be actually used only if fpga component is linked with an app. A test would need to link with it to be included. Thus I added this here.